src/DependencyInjection/Configuration.php

```diff
@@ -12,8 +12,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('bentools_crontab');
+        $treeBuilder = new TreeBuilder('bentools_crontab');
+        $rootNode = $treeBuilder->getRootNode();
 
         $rootNode
```

Comment:
Since SF4 and SF5 share the same structure, and that `%kernel.project_dir%` has been introduced in SF 3.3, I think we could safely set the default value:

```php
        $rootNode
            ->children()
                ->scalarNode('dist_file')
                ->defaultValue('%kernel.project_dir%/config/crontab.dist')
                ->end()
            ->end();
```

So that we could get rid of the `config/packages/bentools_crontab.yaml` file (unless you want to change the crontab file path).

 If so, we could update the README so that the bundle is supposed to work out of the box, just create a `config/crontab.dist` file (the command will yell otherwise, but not the whole DI) and/or create a `config/packages/bentools_crontab.yaml`.

WDYT?

Reply:
I can take care of the doc in a separate PR, I have some minor things to change as well.